extra_scripts/create_imagenet_r_data_files.py

```diff
@@ -0,0 +1,276 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
+
```

Comment:
Is it possible to add a comment explaining what this script is for here?

Reply:
Sure ! I will do this :)